docker/sandbox/site.conf

```diff
@@ -76,7 +77,8 @@ server {
 
     # SUS-5792: rewrite for legacy corporate pages with short article path
     # see wgShortArticlePathWikis MediaWiki variable for the list of them
-    if ($http_x_original_host ~ ^(www|de|fr|pl|es|ja|ru|it).sandbox[^\.]+.wikia.com$) {
+    if ($http_x_original_host ~ "^(www|de|fr|pl|es|ja|ru|it)\.wikia\.com$") {
```

Comment:
This regex was bad - `.` had to be escaped and the entire regex encapsulated inside `"`. In addition, I've added `api.php` specific rule here as there was no `break` in the above rule for `api.php`. Other rules, such as `load.php` and `wikia.php` has `break` so such URLs will never end up here.